modules/Bio/EnsEMBL/IO/Format.pm

```diff
@@ -274,7 +318,7 @@ sub validate_as {
 
 sub validate_as_boolean {
   my ($self, $value) = @_;
-  return ($value == 0 || $value == 1) ? undef : "Value $value is not boolean";;
+  return ($value == 0 || $value == 1) ? 1 : 0;
```

Comment:
I would really prefer using Regexp::Common here instead of reinventing the wheel but this one is so simple that I could live with it.